src/Microsoft.Dnx.Tooling/Program.cs

```diff
@@ -81,7 +81,28 @@ public int Main(string[] args)
             FeedsConsoleCommand.Register(app, reportsFactory);
             ClearCacheConsoleCommand.Register(app, reportsFactory);
 
-            return app.Execute(args);
+            try
+            {
+                return app.Execute(args);
+            }
+            catch (CommandParsingException ex)
+            {
+                AnsiConsole.GetError(useConsoleColor: _runtimeEnv.OperatingSystem == "Windows").WriteLine($"Error: {ex.Message}".Red().Bold());
+                ex.Command.ShowHelp();
+                return 1;
```

Comment:
This is the correct exit code, we'll change all other instances to `1` in #2707.